packages/falso/src/tests/seed.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { rand } from '../lib/core';
+import { seed } from '../lib/random';
+
+describe('seed', () => {
+  it('should return same values with constant seed', () => {
+    const seed1Value = 'constant-seed-1';
+    const seed2Value = 'constant-seed-2';
+    const values = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+    seed(seed1Value);
+    expect(rand(values)).toEqual(1);
```

Comment:
How do you know that value you're going to get?

Reply:
I run it first time, and saw the value. Running the function with same seed should return same value

Reply:
Are we using the fastest algorithm they provide?

Reply:
From `seedrandom` docs, I see that it says that `alea` is fastest 🤔 

https://github.com/davidbau/seedrandom#other-fast-prng-algorithms

Reply:
Hmm, we can start with the current one.